flake.nix

```diff
@@ -45,6 +45,7 @@
         asus-zephyrus-ga402x = import ./asus/zephyrus/ga402x;
         asus-zephyrus-ga402x-amdgpu = import ./asus/zephyrus/ga402x/amdgpu;
         asus-zephyrus-ga402x-nvidia = import ./asus/zephyrus/ga402x/nvidia;
+        asus-zephyrus-ga403 = import ./asus/zephyrus/ga403;
```

Comment:
Please also add this to the README.

Reply:
Done

Reply:
Did you forgot to push? I can't see a change to the table in the toplevel README.md.

Reply:
🤦‍♂️ I misunderstood and thought you wanted it added to flake.nix.It is added to the README table.